### PR TITLE
[Test] Fix test bug

### PR DESCRIPTION
**Pull request by jijoongmoon**
- Add test configuration file in test directory
- modify code to read the path correctly

**Self evaluation:**
1. Build test:	 [X]Passed [ ]Failed [ ]Skipped
2. Run test:	 [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: jijoong.moon <jijoong.moon@samsung.com>

### CODE REVIEW

File: test/tizen_capi/unittest_tizen_capi.cpp [hunk 1]
```diff
@@ -50,7 +50,7 @@ TEST(nntrainer_nnmodel_construct_deconstruct, nntrainer_02_n) {
  * @brief Neural Network Model Construct wit Configuration File Test
  * @return 0 success, -EINVAL if failed.
```
Comment (gichan-jang) [inline test/tizen_capi/unittest_tizen_capi.cpp:51]: Is it right to write the return value here?

File: test/tizen_capi/unittest_tizen_capi.cpp [hunk 2]
```diff
@@ -50,7 +50,7 @@ TEST(nntrainer_nnmodel_construct_deconstruct, nntrainer_02_n) {
...
  */
-TEST(nntrainer_nnmodel_construct_deconstruct, nntrainer_04_n) {
+TEST(nntrainer_nnmodel_construct_deconstruct, nntrainer_03_n) {
```
Comment (gichan-jang) [inline test/tizen_capi/unittest_tizen_capi.cpp:53]: The test name seems to be meaningless.
How about changing it like this?
`TEST(nntrainer_tizen_capi, construct_deconstruct_xx_x) `

Comment (jijoongmoon) [reply test/tizen_capi/unittest_tizen_capi.cpp]: That would be good. Thanks.

File: test/tizen_capi/unittest_tizen_capi.cpp (hunk 1 above)
Comment (jijoongmoon) [reply test/tizen_capi/unittest_tizen_capi.cpp]: removed.

**Pull request merged**